pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -545,6 +549,10 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                         }, outfile)
                         outfile.write('\n')
 
+                    # if it is a new pokemon to our dex, simulate app animation delay
+                    if exp_gain >= 500:
+                        sleep (randrange(newtodex_wait_min, newtodex_wait_max))
```

Comment:
@mjmadsen the variable here should be catchsim_newtodex_wait_min and catchsim_newtodex_wait_max